src/getValueAtZero.js

```diff
@@ -15,8 +15,6 @@
  *
  * @return {function}
  */
-function getValueAtZero(initialFunction) {
-  // write code here
-}
+const getValueAtZero = (initialFunction) => initialFunction(0);
```

Comment:
один параметр не берется в скобки, в этом случае
```suggestion
const getValueAtZero = initialFunction => initialFunction(0);
```